token/transfer-hook/cli/src/main.rs

```diff
@@ -127,6 +128,71 @@ async fn process_create_extra_account_metas(
         .map_err(|err| format!("error: send transaction: {err}").into())
 }
 
+async fn process_update_extra_account_metas(
+    rpc_client: &RpcClient,
+    program_id: &Pubkey,
+    token: &Pubkey,
+    transfer_hook_accounts: Vec<AccountMeta>,
+    mint_authority: &dyn Signer,
+    payer: &dyn Signer,
+) -> Result<Signature, Box<dyn std::error::Error>> {
+    let extra_account_metas_address = get_extra_account_metas_address(token, program_id);
+    let extra_account_metas = transfer_hook_accounts
+        .into_iter()
+        .map(|v| v.into())
+        .collect::<Vec<_>>();
+
+    let length = extra_account_metas.len();
+    let account_size = ExtraAccountMetaList::size_of(length)?;
+    let required_lamports = rpc_client
+        .get_minimum_balance_for_rent_exemption(account_size)
+        .await
+        .map_err(|err| format!("error: unable to fetch rent-exemption: {err}"))?;
+    let extra_account_metas_account = rpc_client.get_account(&extra_account_metas_address).await;
+    if extra_account_metas_account.is_err() {
+        return Err(format!(
+            "error: extra account metas for mint {token} and program {program_id} does not exist"
+        )
+        .into());
+    }
+    let current_lamports = extra_account_metas_account.map(|a| a.lamports).unwrap_or(0);
+    let transfer_lamports = required_lamports.saturating_sub(current_lamports);
```

Comment:
There's a ton of repeat code here compared with the `create` function above it. Can we extract some of this into a helper?